crates/compiler/types/src/subs.rs

```diff
@@ -1754,6 +1756,7 @@ impl Subs {
         symbol_names.push(Symbol::HASH_HASHER);
         symbol_names.push(Symbol::HASH_HASH_ABILITY);
         symbol_names.push(Symbol::BOOL_EQ);
+        symbol_names.push(Symbol::INSPECT_INSPECT_ABILITY);
         // END INIT-SymbolNames
```

Comment:
Curse of IFTTT... we should rename to "INIT-SymbolSubsSlice" ( sorry, I know it's not due to this PR)

Reply:
I don't truly follow, but renamed.